client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -0,0 +1,341 @@
+						_n(
+							'Installing the following plugin: %s',
+							'Installing the following plugins: %s',
+							extensions.length,
+							'woocommerce-admin'
+						),
+						extensionsList
+					) }
+				</Text>
+			</div>
+		);
+	}
+
+	const installingJetpackOrWcShipping =
+		extensions.includes( 'jetpack' ) ||
+		extensions.includes( 'woocommerce-shipping' );
+
+	const accountRequiredText = __(
+		'User accounts are required to use these features.',
+		'woocommerce-admin'
+	);
+	return (
+		<div className="woocommerce-profile-wizard__footnote">
+			<Text variant="caption" as="p">
+				{ sprintf(
+					/* translators: %1$s: a comma separated list of plugins, e.g. Jetpack, Woocommerce Shipping, %2$s: text: 'User accounts are required to use these features.'  */
+					_n(
+						'The following plugin will be installed for free: %1$s. %2$s',
+						'The following plugins will be installed for free: %1$s. %2$s',
+						extensions.length,
+						'woocommerce-admin'
+					),
+					extensionsList,
+					accountRequiredText
+				) }
+			</Text>
+			{ installingJetpackOrWcShipping && (
+				<Text variant="caption" as="p">
+					{ interpolateComponents( {
+						mixedString: __(
+							'By installing Jetpack and WooCommerce Shipping plugins for free you agree to our {{link}}Terms of Service{{/link}}.',
+							'woocommerce-admin'
+						),
+						components: {
+							link: (
+								<Link
+									href="https://wordpress.com/tos/"
+									target="_blank"
+									type="external"
+								/>
+							),
+						},
+					} ) }
+				</Text>
+			) }
+		</div>
+	);
+};
+
+export const SelectiveExtensionsBundle = ( {
+	isInstallingActivating,
+	onSubmit,
+} ) => {
+	const [ showExtensions, setShowExtensions ] = useState( false );
+	const [ values, setValues ] = useState( initialValues );
+
+	return (
+		<div className="woocommerce-profile-wizard__business-details__free-features">
+			<Card>
+				<div className="woocommerce-profile-wizard__business-details__free-features__illustration">
+					<AppIllustration />
+				</div>
+				<div className="woocommerce-admin__business-details__selective-extensions-bundle">
+					<div className="woocommerce-admin__business-details__selective-extensions-bundle__extension">
+						<CheckboxControl
+							id="woocommerce-business-extensions__checkbox"
+							checked={ values.install_extensions }
+							onChange={ ( checked ) => {
+								setValues(
+									setAllPropsToValue( values, checked )
+								);
+							} }
+						/>
+						<p className="woocommerce-admin__business-details__selective-extensions-bundle__description">
+							{ __(
+								'Add recommended business features to my site'
+							) }
+						</p>
+						<Icon
+							className="woocommerce-admin__business-details__selective-extensions-bundle__expand"
+							icon={ showExtensions ? chevronUp : chevronDown }
+							onClick={ () => {
+								const shouldShowExtensions = ! showExtensions;
```

Comment:
The naming is making the logic here a bit unclear IMO

Reply:
@becdetat Yeah not great naming I agree. I'm blanking on something better. `newState`? or `toggleShowExtensions`?

Reply:
Honestly you could just use `! showExtensions` twice and it would be clearer, rather than using an intermediate.

Reply:
ok sure thing.

Reply:
I'm not convinced its clearer, the intermediate is to indicate that we've toggled the boolean value. Anyway, I've made the change here: 4fd2d417dfe58c75f7ed2723699fd0b0aafe959b